src/transformers/models/superpoint/modeling_superpoint.py

```diff
@@ -258,6 +258,9 @@ def _extract_keypoints(self, scores: torch.Tensor) -> Tuple[torch.Tensor, torch.
         # Convert (y, x) to (x, y)
         keypoints = torch.flip(keypoints, [1]).float()
```

Comment:
Do we have somewhere description of the keypoints output format? I mean it would be great to have it in docs and code snippets, and maybe mention it in `post_process_keypoint_detection` method

Reply:
The format is documented in `SuperPointKeypointDescriptionOutput` docstring, do you think I should still add this information somewhere else either way ?